### PR TITLE
Fix readme broken links

### PR DESCRIPTION
**Pull request by manziEric**
update: remove broken links from Readme.md
fix: #517

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,7 +108,6 @@ Search links that point directly to suitable issues to contribute to on GitHub.
 - [8 non-code ways to contribute to open source](https://opensource.com/life/16/1/8-ways-contribute-open-source-without-writing-code) by [OpenSource](https://twitter.com/OpenSourceWay)
 
 ## Using Version Control
-- [Think Like (a) Git](http://think-like-a-git.net/) - Git introduction for "advanced beginners," but are still struggling, in order to give you a simple strategy to safely experiment with git.
```
Comment (erictleung) [inline README.md:112]: Why was this removed? This appears to be just fine.

Comment (manziEric) [reply README.md:112]: @erictleung  yes i see know i was testing this in Firefox and i got The extension HTTPS everywhere and it made the stie look broken because its HTTP. i wil update this pull request

**Review by erictleung — APPROVED** [review]
LGTM! 👍 Thanks for taking the time to fix some of these links.
